lib/modules/manager/html/readme.md

```diff
@@ -1 +1,7 @@
-The `html` manager extracts scripts and CSS includes pointing to cdnjs. It differs from the `cdnjs` manager in that it detects if an integrity hash is already present and updates it too if found.
+The `html` manager updates `<script>` tags and _CSS includes_ that point to the [cdnjs content delivery network](https://cdnjs.com/).
+It also updates Subresource Integrity (SRI) hashes in `integrity` attributes.
+
+Key differences between the `cdnurl` manager and the `html` manager:
```

Comment:
I think the old `cdnjs` manager was renamed/migrated to the [`cdnurl` manager](https://docs.renovatebot.com/modules/manager/cdnurl/)?

We do still have a `cdnjs` _datasource_. Is that what we meant?